chaos_genius/utils/datetime_helper.py

```diff
@@ -16,13 +18,55 @@ def get_rca_date_from_string(date_value):
     return datetime.strptime(date_value, "%Y/%m/%d %H:%M:%S").date()
 
 
-def get_date_string_with_tz(date_value) -> str:
+def get_datetime_string_with_tz(date_value, hourly=False) -> str:
     """Get date string with timezone."""
     tz_offset = SUPPORTED_TIMEZONES[TIMEZONE]
-    main_str = date_value.strftime("%d %b %Y") + f" {TIMEZONE}"
+    if hourly:
+        main_str = date_value.strftime("%d %b %Y %H:%M") + f" {TIMEZONE}"
+    else:
+        main_str = date_value.strftime("%d %b %Y") + f" {TIMEZONE}"
     return f"{main_str} ({tz_offset})" if tz_offset != "" else main_str
 
 
+def _get_tz_from_offset_str(utc_offset_str):
+    # TODO: update code when tz implementation is complete
+    sign = -1 if utc_offset_str[-6] == "-" else 1
```

Comment:
Is it possible that this function is called with an input like `00:00`? If it is, this line will throw an index error.

Reply:
Not possible as the `utc_offset_str` is picked up from our SUPPORTED_TIMEZONES constants.py file.